playbook/build_ami/main.yml

```diff
@@ -112,8 +112,6 @@
     - name: Gather instance metadata
       ec2_metadata_facts:
 
-    - import_tasks: ../appherd/roles/create_superuser/tasks/main.yml
```

Comment:
Dropping this step since it takes 5+ minutes and really only has to happen once in the lifetime of a deployment environment.